### DISCUSSION TITLE
Consider more contrast for comments

### DISCUSSION THREAD

**Discussion by josh-k-cb** [arcticicestudio/nord#242] [Help & Support] [answered]
## Description

The theme could benefit from more color contrast for comments in Typescript files, especially during diff.

Standard comment:
![image](https://user-images.githubusercontent.com/70601887/103059409-073e2180-4573-11eb-8657-8ca8d11fb471.png)

Diff comment:
![image](https://user-images.githubusercontent.com/70601887/103059387-f55c7e80-4572-11eb-81df-a66cc3b1c738.png)

I've personally applied these:
```
  "editor.tokenColorCustomizations": {
    "[Nord]": {
      "comments": "#81A1C1"
    }
  },
  "workbench.colorCustomizations": {
    "[Nord]": {
      "diffEditor.insertedTextBackground": "#5E81AC33"
    }
  }
```

to get this:

Standard comment:
![image](https://user-images.githubusercontent.com/70601887/103059514-55ebbb80-4573-11eb-85b2-0be7395cbddf.png)

Diff comment:
![image](https://user-images.githubusercontent.com/70601887/103059497-4cfaea00-4573-11eb-951c-be7ec2b13e11.png)

My solution might not be 'pure Nord' but I can read comments during diff, which is nice

### Benefits

Readability

### Possible Drawbacks

Objective opinion

**Comment by crizant** [comment]
This is exactly what i've been looking for, thanks!
Finally I can see comments in the git diff panel. I think this should be applied by default.

**Comment by svengreb** [comment, marked as answer]
Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, or being related to the project‘s code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.

---

@josh-k-cb Thanks again for your contributions 🚀
Changes like this can be, like you already did, made by [customizing the theme to fit your needs](https://www.nordtheme.com/docs/ports/visual-studio-code/customization). The [comment color was discussed and adjusted across all syntax highlighting related Nord ports](https://github.com/nordtheme/nord/issues/94) so all details about the decision of the color can be read there.
I‘ll close this discussion since this is a user-side related solution.
